tests/integration/records/single_db/test_records_unload_datetime.py

```diff
@@ -0,0 +1,370 @@
+                                         '.000000+0000\n',
+                                         # TODO: Should this be necessary?
+                                         create_sample(datetimeformattz) +
+                                         f":{SAMPLE_SECOND:02d}.000000\n"
+                                         ],
+                              f"from datetimeformattz {datetimeformattz} and "
+                              f"addl_hints {addl_hints}")
+            except ModuleNotFoundError as e:
+                if 'pandas' in str(e) and expect_pandas_failure:
+                    # as expected
+                    continue
+                else:
+                    raise
+            self.assertFalse(expect_pandas_failure)
+
+    def test_unload_timeonly(self) -> None:
+        self.datetime_fixture.createTimeTable()
+        for timeonlyformat in TIMEONLY_CASES:
+            pandas_compatible_addl_hints: PartialRecordsHints = {
+                'dateformat': 'YYYY-MM-DD',
+                'datetimeformat': f'YYYY-MM-DD {timeonlyformat}',
+                'datetimeformattz': f'YYYY-MM-DD {timeonlyformat}',
+            }
+            addl_hints: PartialRecordsHints = {}
+            uses_pandas = False
+            if self.engine.name == 'redshift' and not self.has_scratch_s3_bucket():
+                uses_pandas = True
+            elif self.engine.name == 'mysql':
+                # mysql has no exporter defined, so everything is via Pandas
+                uses_pandas = True
+            elif self.engine.name == 'postgresql':
+                if timeonlyformat in ['HH:MI:SS', 'HH24:MI:SS']:
+                    # ensure we keep other areas of ISO style
+                    addl_hints = {
+                        'dateformat': 'YYYY-MM-DD',
+                        'datetimeformat': 'YYYY-MM-DD HH:MI:SS',
+                        'datetimeformattz': 'YYYY-MM-DD HH:MI:SSOF',
+                    }
+                else:
+                    uses_pandas = True
+            elif self.engine.name == 'vertica':
+                # Make sure our '\n' strings below are valid when comparing output
+                addl_hints['record-terminator'] = '\n'
+                if timeonlyformat not in ['HH:MI:SS', 'HH24:MI:SS']:
+                    uses_pandas = True
+            elif self.engine.name == 'bigquery':
+                # All current export is via Avro
+                uses_pandas = True
+            if uses_pandas:
+                # we're going to need to be sure to use hints that work in Pandas
+                addl_hints.update(pandas_compatible_addl_hints)
+                if 'AM' in timeonlyformat:
+                    # Pandas is more limited in hint support than it
+                    # should be:
+                    #
+                    # https://github.com/bluelabsio/records-mover/issues/143
+                    # TODO: Should expect failure and complain if it doesn't fail
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker